Travis failure

```
[rviz_visual_tools:cmake] CMake Error at /home/travis/ros/ws_bezier/src/bezier/rviz_visual_tools/CMakeLists.txt:29 (find_package):
[rviz_visual_tools:cmake]   By not providing "FindEigen3.cmake" in CMAKE_MODULE_PATH this project has
[rviz_visual_tools:cmake]   asked CMake to find a package configuration file provided by "Eigen3", but
[rviz_visual_tools:cmake]   CMake did not find one.
[rviz_visual_tools:cmake] 
[rviz_visual_tools:cmake]   Could not find a package configuration file provided by "Eigen3" with any
[rviz_visual_tools:cmake]   of the following names:
[rviz_visual_tools:cmake] 
[rviz_visual_tools:cmake]     Eigen3Config.cmake
[rviz_visual_tools:cmake]     eigen3-config.cmake
[rviz_visual_tools:cmake] 
[rviz_visual_tools:cmake]   Add the installation prefix of "Eigen3" to CMAKE_PREFIX_PATH or set
[rviz_visual_tools:cmake]   "Eigen3_DIR" to a directory containing one of the above files.  If "Eigen3"
[rviz_visual_tools:cmake]   provides a separate development package or SDK, be sure it has been
[rviz_visual_tools:cmake]   installed.
[rviz_visual_tools:cmake] 
[rviz_visual_tools:cmake] 
[rviz_visual_tools:cmake] -- Configuring incomplete, errors occurred!
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,6 +26,7 @@ install:
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
   - git clone https://github.com/InstitutMaupertuis/advanced_industrial_ci.git .advanced_ci_config
   - git clone https://github.com/davetcoleman/rviz_visual_tools.git
+  - sed -i 's/find_package(Eigen3 REQUIRED)//g' rviz_visual_tools/CMakeLists.txt # Remove find_package for Eigen
 
 script:
   - cd .advanced_ci_config && source ./travis_pcl.sh && cd ..

